civis/ml/_model.py

```diff
@@ -192,6 +192,18 @@ def _exception_from_logs(exc, job_id, run_id, client, nlog=15):
     return exc
 
 
+def _get_credential(credential, client=None):
+    """Get the credential ID from either an integer or a string."""
```

Comment:
Oh -- now I remember why I did this. When you post a credential parameter via the API you have to use the ID, not the name.

I'll assume that by the time I'm making the `_create_custom_run` call, I'll have the name of the credential irrespective of how this was instantiated and convert it to an integer.